spec/czCustomizableSpec.js

```diff
@@ -152,6 +152,42 @@ describe('cz-customizable', function() {
     expect(commit).toHaveBeenCalledWith('WIP: this is my work-in-progress');
   });
 
+  it('should allow edit message before commit', function(done) {
+    module.prompter(cz, commit);
+    var commitAnswers = cz.prompt.mostRecentCall.args[1];
+    process.env.EDITOR = 'true';
+
+    var answers = {
+      confirmCommit: 'edit',
+      type: 'feat',
+      subject: 'create a new cool feature'
+    };
+
+    commitAnswers(answers);
+    setTimeout(function() {
+      expect(commit).toHaveBeenCalledWith('feat: create a new cool feature');
+      done();
+    }, 100);
+  });
+
+  it('should not commit if editor returned non-zero value', function(done) {
+    module.prompter(cz, commit);
+    var commitAnswers = cz.prompt.mostRecentCall.args[1];
+    process.env.EDITOR = 'false';
+
+    var answers = {
+      confirmCommit: 'edit',
+      type: 'feat',
+      subject: 'create a new cool feature'
+    };
+
+    commitAnswers(answers);
+    setTimeout(function() {
+      expect(commit.wasCalled).toEqual(false);
```

Comment:
`jasmine-node` api seems slightly different from original `jasmine`.
I have looked up it in `jasmine-node` specs. Haven't found any other documentation.